spring-pulsar-test/src/main/java/org/springframework/pulsar/test/support/PulsarConsumerTestUtil.java

```diff
@@ -53,6 +55,30 @@ public class PulsarConsumerTestUtil<T> implements TopicSpec<T>, SchemaSpec<T>, C
 
 	private List<String> topics;
 
+	private boolean untilMethodAlreadyCalled = false;
+
+	public static <T> TopicSpec<T> consumeMessages() {
+		if (PulsarTestContainerSupport.isContainerStarted()) {
+			return PulsarConsumerTestUtil.consumeMessages(PulsarTestContainerSupport.getPulsarBrokerUrl());
+		}
+		return PulsarConsumerTestUtil.consumeMessages("pulsar://localhost:6650");
+	}
+
+	public static <T> TopicSpec<T> consumeMessages(String url) {
+		Assert.notNull(url, "url must not be null");
+		try {
+			return PulsarConsumerTestUtil.consumeMessages(PulsarClient.builder().serviceUrl(url).build());
```

Comment:
I have a question around this piece of code:
- We are calling `build` on `PulsarClient` once again, so in situations when `PulsarClient` as already been built prior to calling this method would it cause any issues? Or this `build` method is idempotent?

Reply:
The `PulsarClient.builder()` creates a new builder instance and will ultimately construct a new client instance. This is functionally fine but it does bring up a good point. In the cases where we are constructing the client and the consumer factory we need to close them once the util method completes. Let's add that in a subsequent PR. For now, this is fine.

Reply:
Cool, thanks for the explanation!